File: OnlinePK-Android/biz-live/src/main/java/com/netease/biz_live/yunxin/live/model/response/LiveListResponse.java
/*
 * Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 * Use of this source code is governed by a MIT license that can be found in the LICENSE file.
 */

package com.netease.biz_live.yunxin.live.model.response;

import com.netease.biz_live.yunxin.live.model.LiveInfo;

import java.util.List;

/**
 * 直播主页面列表返回值
 */
public class LiveListResponse {
    public int endRow;//int	每页大小
    public boolean hasNextPage;//boolean	是否有下一页
    public boolean hasPreviousPage;//boolean	是否上一页
    public boolean isLastPage;//boolean	最后一页
    public boolean isFirstPage;//boolean	第一页
    public List<LiveInfo> list;//直播房间列表
}

File: OnlinePK-Android/biz-live/src/main/java/com/netease/biz_live/yunxin/live/model/response/LiveListResponse.kt
/*
 *  Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 *  Use of this source code is governed by a MIT license that can be found in the LICENSE file
 */

package com.netease.biz_live.yunxin.live.model.response

import com.netease.yunxin.lib_live_room_service.bean.LiveInfo


/**
 * 直播主页面列表返回值
 */
class LiveListResponse {
    var endRow //int	每页大小
            = 0
    var hasNextPage //boolean	是否有下一页
            = false
    var hasPreviousPage //boolean	是否上一页
            = false
    var isLastPage //boolean	最后一页
            = false
    var isFirstPage //boolean	第一页
            = false
    var list //直播房间列表
            : MutableList<LiveInfo>? = null
}